definitions/marts/looker_studio_marts/ecf/ls_induction_start_date.sqlx

```diff
@@ -9,7 +9,21 @@ config {
     },
     description: "This mart provides a report that gives the induction start/completion date for each FIP ECT participant ID. Currently only the induction completion date is exposed to Lead Providers over the API. This report will help Lead Providers determine where schedule changes are required.",

```

Comment:
It's pedantic but can you make it participant profile ID

Reply:
Also, is this based on ecf_inductions?

Reply:
What is the primary table used to create this mart? Is it participant profiles? Just curious if the lead provider information is based on the school in the latest induction record.